2024-02-12 10:30:12.565 [error] <0.383.0>@lists:reverse:175 CRASH REPORT Process <0.383.0> with 0 neighbours crashed with reason: no function clause matching lists:reverse(#{action => capture,order => 0}) line 175
2024-02-12 10:30:12.566 [error] <0.380.0>@lists:reverse:175 Supervisor {<0.380.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.8>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.382.0>, {<<"client 127.0.0.1:41516 -> 127.0.0.1:5672">>,1}, false, 1000000000) at <0.383.0> exit with reason no function clause matching lists:reverse(#{action => capture,order => 0}) line 175 in context child_terminated
2024-02-12 10:30:12.566 [error] <0.380.0> Supervisor {<0.380.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.8>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.382.0>, {<<"client 127.0.0.1:41516 -> 127.0.0.1:5672">>,1}, false, 1000000000) at <0.383.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 10:31:40.917 [error] <0.397.0>@lists:reverse:175 CRASH REPORT Process <0.397.0> with 0 neighbours crashed with reason: no function clause matching lists:reverse(#{action => capture,order => 0}) line 175
2024-02-12 10:31:40.917 [error] <0.394.0>@lists:reverse:175 Supervisor {<0.394.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.9>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.396.0>, {<<"client 127.0.0.1:41518 -> 127.0.0.1:5672">>,1}, false, 1000000000) at <0.397.0> exit with reason no function clause matching lists:reverse(#{action => capture,order => 0}) line 175 in context child_terminated
2024-02-12 10:31:40.917 [error] <0.394.0> Supervisor {<0.394.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.9>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.396.0>, {<<"client 127.0.0.1:41518 -> 127.0.0.1:5672">>,1}, false, 1000000000) at <0.397.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 10:52:17.795 [error] <0.326.0> Supervisor 'Elixir.Orders.Supervisor' had child 'Elixir.OrdersPipeline' started with 'Elixir.OrdersPipeline':start_link([]) at undefined exit with reason {'EXIT',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"invalid configuration given to Broadway.start_link/2, expected :concurrency to be a keyword list, got: 1">>},[{'Elixir.Broadway',start_link,2,[{file,"lib/broadway.ex"},{line,919}]},{supervisor,do_start_child_i,3,[{file,"supervisor.erl"},{line,414}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,400}]},{supervisor,'-start_children/2-fun-0-',3,[{file,"supervisor.erl"},{line,384}]},{supervisor,...},...]}} in context start_error
2024-02-12 10:52:17.795 [error] <0.324.0> CRASH REPORT Process <0.324.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,'Elixir.OrdersPipeline',{'EXIT',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"invalid configuration given to Broadway.start_link/2, expected :concurrency to be a keyword list, got: 1">>},[{'Elixir.Broadway',start_link,2,[{file,"lib/broadway.ex"},{line,919}]},{supervisor,do_start_child_i,3,[{file,"supervisor.erl"},{line,414}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,400}]},{supervisor,'-start_children/2-fun-0-',...},...]}}}},...} in application_master:init/4 line 142
2024-02-12 11:12:49.101 [error] <0.336.0> CRASH REPORT Process <0.336.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"The module OrdersETS was given as a child to a supervisor\nbut it does not implement child_spec/1.\n\nIf you own the given module, please define a child_spec/1 function\nthat receives an argument and returns a child specification as a map.\nFor example:\n\n    def child_spec(opts) do\n      %{\n        id: __MODULE__,\n        start: {__MODULE__, :start_link, [opts]},\n        type: :worker,\n        restart: :permanent,\n     ...">>} in 'Elixir.Supervisor':init_child/1 line 701 in application_master:init/4 line 142
2024-02-12 12:36:25.792 [error] <0.335.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at undefined exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"invalid options given to BroadwayRabbitMQ.AmqpClient.init/1, unknown options [:arguments], valid options are: [:queue, :connection, :qos, :name, :backoff_min, :backoff_max, :backoff_type, :metadata, :declare, :bindings, :merge_options, :after_connect, :consume_options, :broadway]">>} in 'Elixir.BroadwayRabbitMQ.Producer':'init_client!'/2 line 632 in context start_error
2024-02-12 12:36:25.792 [error] <0.336.0> CRASH REPORT Process <0.336.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"invalid options given to BroadwayRabbitMQ.AmqpClient.init/1, unknown options [:arguments], valid options are: [:queue, :connection, :qos, :name, :backoff_min, :backoff_max, :backoff_type, :metadata, :declare, :bindings, :merge_options, :after_connect, :consume_options, :broadway]">>} in 'Elixir.BroadwayRabbitMQ.Producer':'init_client!'/2 line 632
2024-02-12 12:36:25.792 [error] <0.334.0> Supervisor 'Elixir.OrdersPipeline.Broadway.Supervisor' had child #Ref<0.24874643.2340683779.182205> started with 'Elixir.Supervisor':start_link([#{id => 'Elixir.OrdersPipeline.Broadway.Producer_0',shutdown => 30000,start => {'Elixir.Broadway.Topology.ProducerStage',...}}], [{name,'Elixir.OrdersPipeline.Broadway.ProducerSupervisor'},{strategy,one_for_one},{max_restarts,...},...]) at undefined exit with reason {shutdown,{failed_to_start_child,'Elixir.OrdersPipeline.Broadway.Producer_0',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"invalid options given to BroadwayRabbitMQ.AmqpClient.init/1, unknown options [:arguments], valid options are: [:queue, :connection, :qos, :name, :backoff_min, :backoff_max, :backoff_type, :metadata, :declare, :bindings, :merge_options, :after_connect, :consume_options, :broadway]">>},[{'Elixir.BroadwayRabbitMQ.Producer','init_client!',2,...},...]}}} in context start_error
2024-02-12 12:36:25.792 [error] <0.333.0>@Elixir.Broadway.Topology:init:37 CRASH REPORT Process <0.333.0> with 0 neighbours crashed with reason: no match of right hand value {error,{shutdown,{failed_to_start_child,#Ref<0.24874643.2340683779.182205>,{shutdown,{failed_to_start_child,'Elixir.OrdersPipeline.Broadway.Producer_0',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"invalid options given to BroadwayRabbitMQ.AmqpClient.init/1, unknown options [:arguments], valid options are: [:queue, :connection, :qos, :name, :backoff_min, :backoff_max, :backoff_type, :metadata, :declare, :bindings, :merge_options, :after_connect, :consume_opt...">>},...}}}}}} in 'Elixir.Broadway.Topology':init/1 line 37
2024-02-12 12:36:25.792 [error] <0.332.0>@Elixir.Broadway.Topology:init:37 Supervisor 'Elixir.Orders.Supervisor' had child 'Elixir.OrdersPipeline' started with 'Elixir.OrdersPipeline':start_link([]) at undefined exit with reason no match of right hand value {error,{shutdown,{failed_to_start_child,#Ref<0.24874643.2340683779.182205>,{shutdown,{failed_to_start_child,'Elixir.OrdersPipeline.Broadway.Producer_0',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"invalid options given to BroadwayRabbitMQ.AmqpClient.init/1, unknown options [:arguments], valid options are: [:queue, :connection, :qos, :name, :backoff_min, :backoff_max, :backoff_type, :metadata, :declare, :bindings, :merge_options, :after_connect, :consume_opt...">>},...}}}}}} in 'Elixir.Broadway.Topology':init/1 line 37 in context start_error
2024-02-12 12:36:25.792 [error] <0.330.0> CRASH REPORT Process <0.330.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,'Elixir.OrdersPipeline',{{badmatch,{error,{shutdown,{failed_to_start_child,#Ref<0.24874643.2340683779.182205>,{shutdown,{failed_to_start_child,'Elixir.OrdersPipeline.Broadway.Producer_0',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"invalid options given to BroadwayRabbitMQ.AmqpClient.init/1, unknown options [:arguments], valid options are: [:queue, :connection, :qos, :name, :backoff_min, :backoff_max, :backoff_type, :meta...">>},...}}}}}}},...}}},...} in application_master:init/4 line 142
2024-02-12 12:37:51.865 [error] <0.330.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.350.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.330.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:51.865 [error] <0.330.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.350.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.330.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:51.865 [error] <0.329.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.330.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.350.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.330.0>},70000]}} in context child_terminated
2024-02-12 12:37:51.868 [error] <0.338.0> gen_server <0.338.0> terminated with reason: "stopping because dependent process <0.330.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..."
2024-02-12 12:37:51.868 [error] <0.338.0> CRASH REPORT Process <0.338.0> with 0 neighbours exited with reason: "stopping because dependent process <0.330.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:37:51.869 [error] <0.336.0> Supervisor {<0.336.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.337.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.338.0> exit with reason "stopping because dependent process <0.330.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in context child_terminated
2024-02-12 12:37:51.869 [error] <0.336.0> Supervisor {<0.336.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.337.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.338.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:51.879 [error] <0.352.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.364.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.352.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:51.879 [error] <0.352.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.364.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.352.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:51.879 [error] <0.329.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.352.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.364.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.352.0>},70000]}} in context child_terminated
2024-02-12 12:37:51.880 [error] <0.355.0> gen_server <0.355.0> terminated with reason: "stopping because dependent process <0.352.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..."
2024-02-12 12:37:51.880 [error] <0.355.0> CRASH REPORT Process <0.355.0> with 0 neighbours exited with reason: "stopping because dependent process <0.352.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:37:51.881 [error] <0.353.0> Supervisor {<0.353.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.354.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.355.0> exit with reason "stopping because dependent process <0.352.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in context child_terminated
2024-02-12 12:37:51.881 [error] <0.353.0> Supervisor {<0.353.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.354.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.355.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:51.888 [error] <0.366.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.378.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.366.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:51.888 [error] <0.366.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.378.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.366.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:51.888 [error] <0.329.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.366.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.378.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.366.0>},70000]}} in context child_terminated
2024-02-12 12:37:51.888 [error] <0.329.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.366.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:51.888 [error] <0.328.0> Supervisor 'Elixir.OrdersPipeline.Broadway.Supervisor' had child #Ref<0.2084626937.3951558657.213295> started with 'Elixir.Supervisor':start_link([#{id => 'Elixir.OrdersPipeline.Broadway.Producer_0',shutdown => 30000,start => {'Elixir.Broadway.Topology.ProducerStage',...}}], [{name,'Elixir.OrdersPipeline.Broadway.ProducerSupervisor'},{strategy,one_for_one},{max_restarts,...},...]) at <0.329.0> exit with reason shutdown in context child_terminated
2024-02-12 12:37:51.889 [error] <0.369.0> gen_server <0.369.0> terminated with reason: "stopping because dependent process <0.366.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..."
2024-02-12 12:37:51.889 [error] <0.369.0> CRASH REPORT Process <0.369.0> with 0 neighbours exited with reason: "stopping because dependent process <0.366.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:37:51.889 [error] <0.367.0> Supervisor {<0.367.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.368.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.369.0> exit with reason "stopping because dependent process <0.366.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in context child_terminated
2024-02-12 12:37:51.889 [error] <0.367.0> Supervisor {<0.367.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.368.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.369.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:51.895 [error] <0.381.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.397.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.381.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:51.895 [error] <0.381.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.397.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.381.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:51.895 [error] <0.380.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.381.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.397.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.381.0>},70000]}} in context child_terminated
2024-02-12 12:37:51.895 [error] <0.388.0> gen_server <0.388.0> terminated with reason: "stopping because dependent process <0.381.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..."
2024-02-12 12:37:51.896 [error] <0.388.0> CRASH REPORT Process <0.388.0> with 0 neighbours exited with reason: "stopping because dependent process <0.381.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:37:51.896 [error] <0.386.0> Supervisor {<0.386.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.387.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.388.0> exit with reason "stopping because dependent process <0.381.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in context child_terminated
2024-02-12 12:37:51.896 [error] <0.386.0> Supervisor {<0.386.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.387.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.388.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:51.902 [error] <0.399.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.411.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.399.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:51.971 [error] <0.399.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.411.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.399.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:51.971 [error] <0.380.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.399.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.411.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.399.0>},70000]}} in context child_terminated
2024-02-12 12:37:51.972 [error] <0.402.0> gen_server <0.402.0> terminated with reason: "stopping because dependent process <0.399.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..."
2024-02-12 12:37:52.004 [error] <0.618.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.634.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.618.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.006 [error] <0.618.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.634.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.618.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.006 [error] <0.617.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.618.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.634.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.618.0>},70000]}} in context child_terminated
2024-02-12 12:37:52.007 [error] <0.624.0> gen_server <0.624.0> terminated with reason: "stopping because dependent process <0.618.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..."
2024-02-12 12:37:52.007 [error] <0.624.0> CRASH REPORT Process <0.624.0> with 0 neighbours exited with reason: "stopping because dependent process <0.618.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:37:52.007 [error] <0.622.0> Supervisor {<0.622.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.623.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.624.0> exit with reason "stopping because dependent process <0.618.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in context child_terminated
2024-02-12 12:37:52.007 [error] <0.622.0> Supervisor {<0.622.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.623.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.624.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:52.013 [error] <0.636.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.648.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.636.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.013 [error] <0.636.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.648.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.636.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.013 [error] <0.617.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.636.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.648.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.636.0>},70000]}} in context child_terminated
2024-02-12 12:37:52.013 [error] <0.639.0> gen_server <0.639.0> terminated with reason: "stopping because dependent process <0.636.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..."
2024-02-12 12:37:52.014 [error] <0.639.0> CRASH REPORT Process <0.639.0> with 0 neighbours exited with reason: "stopping because dependent process <0.636.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:37:52.014 [error] <0.637.0> Supervisor {<0.637.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.638.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.639.0> exit with reason "stopping because dependent process <0.636.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in context child_terminated
2024-02-12 12:37:52.014 [error] <0.637.0> Supervisor {<0.637.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.638.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.639.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:52.023 [error] <0.650.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.662.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.650.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.024 [error] <0.650.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.662.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.650.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.024 [error] <0.617.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.650.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.662.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.650.0>},70000]}} in context child_terminated
2024-02-12 12:37:52.024 [error] <0.617.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.650.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:52.024 [error] <0.522.0> Supervisor 'Elixir.OrdersPipeline.Broadway.Supervisor' had child #Ref<0.2084626937.3951558661.211871> started with 'Elixir.Supervisor':start_link([#{id => 'Elixir.OrdersPipeline.Broadway.Producer_0',shutdown => 30000,start => {'Elixir.Broadway.Topology.ProducerStage',...}}], [{name,'Elixir.OrdersPipeline.Broadway.ProducerSupervisor'},{strategy,one_for_one},{max_restarts,...},...]) at <0.617.0> exit with reason shutdown in context child_terminated
2024-02-12 12:37:52.025 [error] <0.653.0> gen_server <0.653.0> terminated with reason: "stopping because dependent process <0.650.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..."
2024-02-12 12:37:52.025 [error] <0.653.0> CRASH REPORT Process <0.653.0> with 0 neighbours exited with reason: "stopping because dependent process <0.650.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:37:52.025 [error] <0.651.0> Supervisor {<0.651.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.652.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.653.0> exit with reason "stopping because dependent process <0.650.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in context child_terminated
2024-02-12 12:37:52.025 [error] <0.651.0> Supervisor {<0.651.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.652.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.653.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:52.031 [error] <0.665.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.681.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.665.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.031 [error] <0.665.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.681.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.665.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.031 [error] <0.664.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.665.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.681.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.665.0>},70000]}} in context child_terminated
2024-02-12 12:37:52.032 [error] <0.671.0> gen_server <0.671.0> terminated with reason: "stopping because dependent process <0.665.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..."
2024-02-12 12:37:52.033 [error] <0.671.0> CRASH REPORT Process <0.671.0> with 0 neighbours exited with reason: "stopping because dependent process <0.665.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:37:52.034 [error] <0.668.0> Supervisor {<0.668.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.669.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.671.0> exit with reason "stopping because dependent process <0.665.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in context child_terminated
2024-02-12 12:37:52.034 [error] <0.668.0> Supervisor {<0.668.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.669.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.671.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:52.039 [error] <0.683.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.695.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.683.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.115 [error] <0.683.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.695.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.683.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.115 [error] <0.664.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.683.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.695.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.683.0>},70000]}} in context child_terminated
2024-02-12 12:37:52.116 [error] <0.686.0> gen_server <0.686.0> terminated with reason: "stopping because dependent process <0.683.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..."
2024-02-12 12:37:52.116 [error] <0.686.0> CRASH REPORT Process <0.686.0> with 0 neighbours exited with reason: "stopping because dependent process <0.683.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:37:52.116 [error] <0.684.0> Supervisor {<0.684.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.685.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.686.0> exit with reason "stopping because dependent process <0.683.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in context child_terminated
2024-02-12 12:37:52.116 [error] <0.684.0> Supervisor {<0.684.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.685.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.686.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:52.116 [error] <0.697.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.709.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.697.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.117 [error] <0.697.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.709.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.697.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.117 [error] <0.664.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.697.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.709.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.697.0>},70000]}} in context child_terminated
2024-02-12 12:37:52.117 [error] <0.664.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.697.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:52.117 [error] <0.522.0> Supervisor 'Elixir.OrdersPipeline.Broadway.Supervisor' had child #Ref<0.2084626937.3951558661.211871> started with 'Elixir.Supervisor':start_link([#{id => 'Elixir.OrdersPipeline.Broadway.Producer_0',shutdown => 30000,start => {'Elixir.Broadway.Topology.ProducerStage',...}}], [{name,'Elixir.OrdersPipeline.Broadway.ProducerSupervisor'},{strategy,one_for_one},{max_restarts,...},...]) at <0.664.0> exit with reason shutdown in context child_terminated
2024-02-12 12:37:52.117 [error] <0.522.0> Supervisor 'Elixir.OrdersPipeline.Broadway.Supervisor' had child #Ref<0.2084626937.3951558661.211871> started with 'Elixir.Supervisor':start_link([#{id => 'Elixir.OrdersPipeline.Broadway.Producer_0',shutdown => 30000,start => {'Elixir.Broadway.Topology.ProducerStage',...}}], [{name,'Elixir.OrdersPipeline.Broadway.ProducerSupervisor'},{strategy,one_for_one},{max_restarts,...},...]) at <0.664.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:52.117 [error] <0.326.0> Supervisor 'Elixir.Orders.Supervisor' had child 'Elixir.OrdersPipeline' started with 'Elixir.OrdersPipeline':start_link([]) at <0.521.0> exit with reason shutdown in context child_terminated
2024-02-12 12:37:52.118 [error] <0.700.0> gen_server <0.700.0> terminated with reason: "stopping because dependent process <0.697.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..."
2024-02-12 12:37:52.118 [error] <0.700.0> CRASH REPORT Process <0.700.0> with 0 neighbours exited with reason: "stopping because dependent process <0.697.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:37:52.118 [error] <0.698.0> Supervisor {<0.698.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.699.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.700.0> exit with reason "stopping because dependent process <0.697.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none\">>}},\n                                                    {gen_server,call,\n       ..." in context child_terminated
2024-02-12 12:37:52.118 [error] <0.698.0> Supervisor {<0.698.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.699.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.700.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:37:52.118 [error] <0.714.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.730.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.714.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.118 [error] <0.714.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.730.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.714.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:37:52.119 [error] <0.713.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.714.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders_queue' in vhost '/': received the value '5000' of type 'long' but current is none">>}},{gen_server,call,[<0.730.0>,{call,{'queue.declare',0,<<"orders_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,5000}]},none,<0.714.0>},70000]}} in context child_terminated
2024-02-12 12:39:33.368 [error] <0.330.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.350.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.330.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:39:33.369 [error] <0.330.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.350.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.330.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:39:33.369 [error] <0.329.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.330.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.350.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.330.0>},70000]}} in context child_terminated
2024-02-12 12:39:33.371 [error] <0.338.0> gen_server <0.338.0> terminated with reason: "stopping because dependent process <0.330.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..."
2024-02-12 12:39:33.374 [error] <0.338.0> CRASH REPORT Process <0.338.0> with 0 neighbours exited with reason: "stopping because dependent process <0.330.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:39:33.374 [error] <0.336.0> Supervisor {<0.336.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.337.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.338.0> exit with reason "stopping because dependent process <0.330.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..." in context child_terminated
2024-02-12 12:39:33.374 [error] <0.336.0> Supervisor {<0.336.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.337.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.338.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:39:33.379 [error] <0.352.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.364.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.352.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:39:33.379 [error] <0.352.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.364.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.352.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:39:33.379 [error] <0.329.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.352.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.364.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.352.0>},70000]}} in context child_terminated
2024-02-12 12:39:33.379 [error] <0.355.0> gen_server <0.355.0> terminated with reason: "stopping because dependent process <0.352.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..."
2024-02-12 12:39:33.380 [error] <0.355.0> CRASH REPORT Process <0.355.0> with 0 neighbours exited with reason: "stopping because dependent process <0.352.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:39:33.380 [error] <0.353.0> Supervisor {<0.353.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.354.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.355.0> exit with reason "stopping because dependent process <0.352.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..." in context child_terminated
2024-02-12 12:39:33.380 [error] <0.353.0> Supervisor {<0.353.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.354.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.355.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:39:33.387 [error] <0.366.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.378.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.366.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:39:33.387 [error] <0.366.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.378.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.366.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:39:33.387 [error] <0.329.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.366.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.378.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.366.0>},70000]}} in context child_terminated
2024-02-12 12:39:33.387 [error] <0.329.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.366.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:39:33.387 [error] <0.328.0> Supervisor 'Elixir.OrdersPipeline.Broadway.Supervisor' had child #Ref<0.186241234.1266941956.210738> started with 'Elixir.Supervisor':start_link([#{id => 'Elixir.OrdersPipeline.Broadway.Producer_0',shutdown => 30000,start => {'Elixir.Broadway.Topology.ProducerStage',...}}], [{name,'Elixir.OrdersPipeline.Broadway.ProducerSupervisor'},{strategy,one_for_one},{max_restarts,...},...]) at <0.329.0> exit with reason shutdown in context child_terminated
2024-02-12 12:39:33.388 [error] <0.369.0> gen_server <0.369.0> terminated with reason: "stopping because dependent process <0.366.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..."
2024-02-12 12:39:33.388 [error] <0.369.0> CRASH REPORT Process <0.369.0> with 0 neighbours exited with reason: "stopping because dependent process <0.366.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:39:33.388 [error] <0.367.0> Supervisor {<0.367.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.368.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.369.0> exit with reason "stopping because dependent process <0.366.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..." in context child_terminated
2024-02-12 12:39:33.388 [error] <0.367.0> Supervisor {<0.367.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.368.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.369.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:39:33.397 [error] <0.381.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.397.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.381.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:39:33.397 [error] <0.381.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.397.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.381.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:39:33.397 [error] <0.380.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.381.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.397.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.381.0>},70000]}} in context child_terminated
2024-02-12 12:39:33.399 [error] <0.388.0> gen_server <0.388.0> terminated with reason: "stopping because dependent process <0.381.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..."
2024-02-12 12:39:33.399 [error] <0.388.0> CRASH REPORT Process <0.388.0> with 0 neighbours exited with reason: "stopping because dependent process <0.381.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:39:33.399 [error] <0.386.0> Supervisor {<0.386.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.387.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.388.0> exit with reason "stopping because dependent process <0.381.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..." in context child_terminated
2024-02-12 12:39:33.399 [error] <0.386.0> Supervisor {<0.386.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.387.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.388.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:39:33.406 [error] <0.399.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.411.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.399.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:39:33.488 [error] <0.399.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.411.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.399.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:39:33.489 [error] <0.380.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.399.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'">>}},{gen_server,call,[<0.411.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[{<<"x-message-ttl">>,long,15000}]},none,<0.399.0>},70000]}} in context child_terminated
2024-02-12 12:39:33.489 [error] <0.402.0> gen_server <0.402.0> terminated with reason: "stopping because dependent process <0.399.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received '15000' but current is '5000'\">>}},\n                                                    {gen_server,call,\n                            ..."
2024-02-12 12:40:10.139 [error] <0.330.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.350.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.330.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:40:10.140 [error] <0.330.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.350.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.330.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:40:10.140 [error] <0.329.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.330.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.350.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.330.0>},70000]}} in context child_terminated
2024-02-12 12:40:10.142 [error] <0.338.0> gen_server <0.338.0> terminated with reason: "stopping because dependent process <0.330.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..."
2024-02-12 12:40:10.142 [error] <0.338.0> CRASH REPORT Process <0.338.0> with 0 neighbours exited with reason: "stopping because dependent process <0.330.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:40:10.142 [error] <0.336.0> Supervisor {<0.336.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.337.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.338.0> exit with reason "stopping because dependent process <0.330.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..." in context child_terminated
2024-02-12 12:40:10.142 [error] <0.336.0> Supervisor {<0.336.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.337.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.338.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:40:10.148 [error] <0.352.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.364.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.352.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:40:10.149 [error] <0.352.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.364.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.352.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:40:10.149 [error] <0.329.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.352.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.364.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.352.0>},70000]}} in context child_terminated
2024-02-12 12:40:10.149 [error] <0.355.0> gen_server <0.355.0> terminated with reason: "stopping because dependent process <0.352.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..."
2024-02-12 12:40:10.150 [error] <0.355.0> CRASH REPORT Process <0.355.0> with 0 neighbours exited with reason: "stopping because dependent process <0.352.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:40:10.150 [error] <0.353.0> Supervisor {<0.353.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.354.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.355.0> exit with reason "stopping because dependent process <0.352.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..." in context child_terminated
2024-02-12 12:40:10.150 [error] <0.353.0> Supervisor {<0.353.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.354.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.355.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:40:10.155 [error] <0.366.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.378.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.366.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:40:10.156 [error] <0.366.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.378.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.366.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:40:10.156 [error] <0.329.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.366.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.378.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.366.0>},70000]}} in context child_terminated
2024-02-12 12:40:10.156 [error] <0.329.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.366.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:40:10.156 [error] <0.328.0> Supervisor 'Elixir.OrdersPipeline.Broadway.Supervisor' had child #Ref<0.948930215.2609119236.81623> started with 'Elixir.Supervisor':start_link([#{id => 'Elixir.OrdersPipeline.Broadway.Producer_0',shutdown => 30000,start => {'Elixir.Broadway.Topology.ProducerStage',...}}], [{name,'Elixir.OrdersPipeline.Broadway.ProducerSupervisor'},{strategy,one_for_one},{max_restarts,...},...]) at <0.329.0> exit with reason shutdown in context child_terminated
2024-02-12 12:40:10.156 [error] <0.369.0> gen_server <0.369.0> terminated with reason: "stopping because dependent process <0.366.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..."
2024-02-12 12:40:10.156 [error] <0.369.0> CRASH REPORT Process <0.369.0> with 0 neighbours exited with reason: "stopping because dependent process <0.366.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:40:10.156 [error] <0.367.0> Supervisor {<0.367.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.368.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.369.0> exit with reason "stopping because dependent process <0.366.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..." in context child_terminated
2024-02-12 12:40:10.156 [error] <0.367.0> Supervisor {<0.367.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.368.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.369.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:40:10.162 [error] <0.381.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.397.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.381.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:40:10.162 [error] <0.381.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.397.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.381.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:40:10.163 [error] <0.380.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.381.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.397.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.381.0>},70000]}} in context child_terminated
2024-02-12 12:40:10.163 [error] <0.388.0> gen_server <0.388.0> terminated with reason: "stopping because dependent process <0.381.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..."
2024-02-12 12:40:10.163 [error] <0.388.0> CRASH REPORT Process <0.388.0> with 0 neighbours exited with reason: "stopping because dependent process <0.381.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:40:10.163 [error] <0.384.0> Supervisor {<0.384.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.385.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.388.0> exit with reason "stopping because dependent process <0.381.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..." in context child_terminated
2024-02-12 12:40:10.163 [error] <0.384.0> Supervisor {<0.384.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.385.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.388.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:40:10.169 [error] <0.399.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.411.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.399.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:40:10.255 [error] <0.399.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.411.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.399.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:40:10.256 [error] <0.380.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.399.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.411.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.399.0>},70000]}} in context child_terminated
2024-02-12 12:40:10.257 [error] <0.402.0> gen_server <0.402.0> terminated with reason: "stopping because dependent process <0.399.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..."
2024-02-12 12:40:10.257 [error] <0.402.0> CRASH REPORT Process <0.402.0> with 0 neighbours exited with reason: "stopping because dependent process <0.399.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:40:10.257 [error] <0.400.0> Supervisor {<0.400.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.401.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.402.0> exit with reason "stopping because dependent process <0.399.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..." in context child_terminated
2024-02-12 12:40:10.258 [error] <0.400.0> Supervisor {<0.400.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.401.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.402.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:40:10.259 [error] <0.413.0> gen_server 'Elixir.OrdersPipeline.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.425.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.413.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:40:10.259 [error] <0.413.0> CRASH REPORT Process 'Elixir.OrdersPipeline.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.425.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.413.0>},70000]}} in gen_server:call/3 line 382
2024-02-12 12:40:10.259 [error] <0.380.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.413.0> exit with reason {{shutdown,{server_initiated_close,406,<<"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'">>}},{gen_server,call,[<0.425.0>,{call,{'queue.declare',0,<<"orders2_queue">>,false,true,false,false,false,[]},none,<0.413.0>},70000]}} in context child_terminated
2024-02-12 12:40:10.259 [error] <0.380.0> Supervisor 'Elixir.OrdersPipeline.Broadway.ProducerSupervisor' had child 'Elixir.OrdersPipeline.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{batchers,[]},{context,context_not_set},{resubscribe_interval,...},...]},...], 0, [{name,'Elixir.OrdersPipeline.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.413.0> exit with reason reached_max_restart_intensity in context shutdown
2024-02-12 12:40:10.260 [error] <0.328.0> Supervisor 'Elixir.OrdersPipeline.Broadway.Supervisor' had child #Ref<0.948930215.2609119236.81623> started with 'Elixir.Supervisor':start_link([#{id => 'Elixir.OrdersPipeline.Broadway.Producer_0',shutdown => 30000,start => {'Elixir.Broadway.Topology.ProducerStage',...}}], [{name,'Elixir.OrdersPipeline.Broadway.ProducerSupervisor'},{strategy,one_for_one},{max_restarts,...},...]) at <0.380.0> exit with reason shutdown in context child_terminated
2024-02-12 12:40:10.260 [error] <0.416.0> gen_server <0.416.0> terminated with reason: "stopping because dependent process <0.413.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..."
2024-02-12 12:40:10.260 [error] <0.416.0> CRASH REPORT Process <0.416.0> with 0 neighbours exited with reason: "stopping because dependent process <0.413.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..." in gen_server:handle_common_reply/8 line 1238
2024-02-12 12:40:10.261 [error] <0.414.0> Supervisor {<0.414.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.415.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.416.0> exit with reason "stopping because dependent process <0.413.0> died: {{shutdown,\n                                                     {server_initiated_close,\n                                                      406,\n                                                      <<\"PRECONDITION_FAILED - inequivalent arg 'x-message-ttl' for queue 'orders2_queue' in vhost '/': received none but current is the value '5000' of type 'long'\">>}},\n                                                    {gen_server,call,\n      ..." in context child_terminated
